test/test_multiprocessing.py

```diff
@@ -325,7 +325,7 @@ def test_cuda_small_tensors(self):
             self.assertEqual(tensor_size, 5)
             self.assertEqual(storage_size, 5)
 
-    @unittest.skipIf(IS_WINDOWS, 'NYI: not supported on Windows')
+    # @unittest.skipIf(IS_WINDOWS, 'NYI: not supported on Windows')
```

Comment:
I don't know why this one should throw exception in Unix systems.

Reply:
Because it forks with a CUDA context already initialized. Does it raise on Windows as well? We should either leave the decoration or remove it altogether

Reply:
No, it doesn't throw exceptions here. Since it's about process forking which has nothing to do with Windows, I'm thinking of leave the decoration there.

Reply:
I think leaving it is a better idea, but the message should be changed saying that it doesn't apply to windows, because it only fails when using `fork`